src/Redist/Microsoft.ML.TensorFlow.Redist/Microsoft.ML.Tensorflow.Redist.proj

```diff
@@ -0,0 +1,133 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Restore;Build">
```

Comment:
Do you really need `Restore`?  I don't see any `PackageReference`

Reply:
SDK has a hard dependency.  There's an implicit package reference, but even when I disabled it the SDK still complained.

Reply:
SDK has a hard dependency.  There's an implicit package reference, but even when I disabled it the SDK still complained.

---
In reply to: [212420522](https://github.com/dotnet/machinelearning/pull/720#discussion_r212420522) [](ancestors = 212420522)